src/filters/price_filter.rs

```diff
@@ -0,0 +1,79 @@
+use super::*;
+
+pub struct PriceFilter {
+    pub locale: String,
+}
+
+impl PriceFilter {
+    fn currencies(&self) -> &'static str {
+        "$¢£¤¥֏؋৲৳৻૱௹฿₠₡₢₣₤₥₦₧₨₩₪₫€₭₮₯₰₱₲₳₴₵₶₷₸₹₺₽﹩＄￠￡￥￦"
+    }
+}
+
+impl LinterFilter for PriceFilter {
+    fn locales(&self) -> Vec<&'static str> {
+        vec!("fr")
+    }
+
+    fn message(&self) -> &'static str {
+        match &self.locale as &str {
+            "fr" => "The currency sign should be written after the amount and a non-breaking space.",
+            _ => "",
+        }
+    }
+
+    fn regex_pattern(&self) -> String {
+        match &self.locale as &str {
+            "fr" => format!("([\\d]+[^ ][{}]{{1}}|[{}]{{1}}[\\s]?[\\d]+)", self.currencies(), self.currencies()),
```

Comment:
For future developers sanity, I think this line would benefit from a full-line comment above it, showing example of what it will produce (even parts of it).